src/cmdc_tools/datasets/nytimes/data.py

```diff
@@ -29,7 +30,7 @@ def _insert_query(self, df: pd.DataFrame, table_name: str, temp_name: str, pk: s
 
     def get(self):
         df = pd.read_csv(self.url, parse_dates=["date"])
-        df["vintage"] = pd.datetime.today().date()
+        df["vintage"] = self._retrieve_vintage()
```

Comment:
I changed this to use the `_retrieve_vintage` method and moved the vintage methods to the `DatasetBase` class

Reply:
ok thanks